### ISSUE TITLE
Labels being created multiple times

### ISSUE THREAD

**Issue by davidbren** [explosion/spaCy#3130]
## How to reproduce the behaviour
Created a custom NER model the cfg file is showing my entities multiple times 

"extra_labels":[
    "PAY",
    "PAY",
    "PAY",
    "PAY",
    "CPTI",
    "CPTI",
    "CPTI",
    "CPTI",
    "CHG_Amts",
    "CHG_Amts",
    "CHG_Amts",
    "CHG_Amts"
  ],

## Your Environment

* Operating System: Windows 7
* Python Version Used:  3.6
* spaCy Version Used: 2.0.18
* Environment Information:

**Comment by rbhambriiit** [comment]
I suspect that This might be due to your data format of list of list as you showed in#3120.

Can you try with list of tuple to see if it helps.

Otherwise share more details on the code you run to get this

**Comment by davidbren** [comment]
My training set is a local json file to read it in I deserialize using the following 

    with open(localFilePath,encoding='utf-8') as f:
         data = json.load(f)

 If I convert to a list of tuples this fails

I re-checked the code  where I add the labels 
        for _, annotations in trainData:
            for ent in annotations.get('entities'):
                ner.add_label(ent[2])

ent[2] = 'PAY' so this seems correct ? 
Have you any examples of a local json file of training data ?

**Comment by ines** [comment]
Looks like this is related to #2179. It should be fixed in the latest `spacy-nightly` already – you can install the latest release from pip and rerun your code to check 🙂

**Comment by davidbren** [comment]
I don't know I am thinking that maybe their is something wrong with my representation as the issue in  #3120, seems to suggest their is something going on. 

By any chance have you any examples of a local file of training data being deserialized, it would really help, even better if you have any that are compatible with json.load()

**Comment by davidbren** [comment]
I reworked the format changing the lists to tuples & it made no difference
